node/overseer/orchestra/proc-macro/src/impl_builder.rs

```diff
@@ -83,7 +83,7 @@ pub(crate) fn impl_builder(info: &OverseerInfo) -> proc_macro2::TokenStream {
 
 	// Helpers to use within quote! macros
 	let spawner_where_clause: syn::TypeParam = parse_quote! {
-			S: #support_crate ::SpawnNamed + Send
+			S: #support_crate ::Spawner
```

Comment:
So we have finally removed that `Send` requirement.

Reply:
It's implicitly required by the `Spawner` trait already.